ui/app/js/controllers/user_edit_controller.js

```diff
@@ -100,11 +100,14 @@ function ($scope, $modalInstance, CSRF, Permissions, user, users) {
           role: '',
           data_version: ''
         };
+        $scope.errors = {
+          roles: {}
```

Comment:
Have you had a look at the failing tests? Sometimes they need updates when you change related code: https://github.com/mozilla/balrog/blob/master/ui/spec/controllers/user_edit_controller_spec.js